src/actions/userActions.js

```diff
@@ -43,3 +43,20 @@ export function addUser(user) {
     });
   }
 }
+
+export function addUserFeed(userId, feedback, timestamp) {
+  return dispatch => {
+    const url = `/users/feed/`;
```

Comment:
Did you mean to put the feed endpoint under user? I think for now we can do `/users/<user id>/feed`.